safe/gui/widgets/dock.py

```diff
@@ -1742,7 +1752,14 @@ def layer_changed(self, layer):
         try:
             keywords = self.keyword_io.read_keywords(layer)
 
-            if keywords.get('layer_purpose') == 'impact':
+            # list of layer purpose to show impact report
+            impacted_layer = [
+                'impact',
+                'analysis',
+                'exposure_breakdown',
+            ]
```

Comment:
Will you show a report for each of these layers ? Or you mean you need these layers to build the impact report ?